numpy/lib/type_check.py

```diff
@@ -467,7 +467,10 @@ def real_if_close(a,tol=100):
 
 def asscalar(a):
     """
-    Convert an array of size 1 to its scalar equivalent.
+    Convert an array of size 1 to its scalar equivalent. Deprecated, use
+    `numpy.ndarray.item()` instead.
```

Comment:
This second sentence should be indented under the `.. deprecated` block

Reply:
moved